docs/_includes/generated/netty_shuffle_environment_configuration.html

```diff
@@ -57,17 +57,23 @@
             <td>Number of extra network buffers to use for each outgoing/incoming gate (result partition/input gate). In credit-based flow control mode, this indicates how many floating credits are shared among all the input channels. The floating buffers are distributed based on backlog (real-time output buffers in the subpartition) feedback, and can help relieve back-pressure caused by unbalanced data distribution among the subpartitions. This value should be increased in case of higher round trip times between nodes and/or larger number of machines in the cluster.</td>
         </tr>
         <tr>
-            <td><h5>taskmanager.network.netty.client.connectTimeoutSec</h5></td>
-            <td style="word-wrap: break-word;">120</td>
+            <td><h5>taskmanager.network.retries</h5></td>
+            <td style="word-wrap: break-word;">0</td>
             <td>Integer</td>
-            <td>The Netty client connection timeout.</td>
+            <td>The number of retry attempts for network communication. Currently it's only used for establishing input/output channel connections</td>
         </tr>
         <tr>
             <td><h5>taskmanager.network.netty.client.numThreads</h5></td>
             <td style="word-wrap: break-word;">-1</td>
             <td>Integer</td>
             <td>The number of Netty client threads.</td>
         </tr>
+        <tr>
+            <td><h5>taskmanager.network.netty.clientRetry</h5></td>
```

Comment:
The generated key is not consistent with the new added "taskmanager.network.retries" in `NettyShuffleEnvironmentOptions`. Do you adjust these files manually or automatically?

Reply:
It is a mistake. I manually changed the file. I will change it back.